pkg/localize/locales/en/cmd/kafka_consumergroup_reset_offset.toml

```
@@ -0,0 +1,43 @@
+[kafka.consumerGroup.resetOffset.cmd.shortDescription]
+one = 'Reset offset of a consumer group'
+
+[kafka.consumerGroup.resetOffset.cmd.longDescription]
+one = '''
+Reset offset for a consumer group from the Kafka instance.
```

```suggestion
Reset the offset for a consumer group.
```
I think the fact that this applies to a Kafka instance should be clear without stating it.